driver-core/src/test/functional/com/mongodb/ClusterFixture.java

```diff
@@ -143,6 +144,47 @@ public static boolean serverVersionGreaterThan(final String versionString) {
         return getServerVersion().compareTo(new ServerVersion(getVersionList(versionString).subList(0, 3))) > 0;
     }
 
+    public static boolean canRunTests(final BsonArray runOn) {
+        boolean topologyFound = false;
+        ServerVersion serverVersion = getServerVersion();
+        for (BsonValue info : runOn) {
+            final BsonDocument document = info.asDocument();
+
+            if (document.containsKey("minServerVersion") && serverVersion.compareTo(getServerVersion("minServerVersion", document)) < 0) {
+                continue;
+            }
+            if (document.containsKey("maxServerVersion") && serverVersion.compareTo(getServerVersion("maxServerVersion", document)) < 0) {
```

Comment:
I think this should be `> 0` if the intention is to skip the test when the actual server version is greater than the maxServerVersion

Reply:
Good catch :)